packages/lib/server-only/auth/send-confirmation-email.ts

```diff
@@ -0,0 +1,56 @@
+import { createElement } from 'react';
+
+import { mailer } from '@documenso/email/mailer';
+import { render } from '@documenso/email/render';
```

Comment:
You may want to use `renderAsync` if you're getting the `renderToReadableStream`